### ISSUE TITLE
INDIRECT calculation problem

### ISSUE THREAD

**Issue by loyating** [PHPOffice/PhpSpreadsheet#414]
This is:

- [x] bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)

### What is the expected behavior?
The formula can be parsed normally

### What is the current behavior?
INDIRECT formula parsing error

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```
Create an Excel cell to set the formula (for example): =INDIRECT("R"&(ROW($A90)-1)&"C"&(COLUMN($A90)),0)


Fatal error: Uncaught TypeError: Argument 2 passed to PhpOffice\PhpSpreadsheet\Calculation\LookupRef::INDIRECT() must be an instance of PhpOffice\PhpSpreadsheet\Cell\Cell, integer given in /vendor/phpoffice /phpspreadsheet/src/PhpSpreadsheet/Calculation/LookupRef.php:280
Stack trace:
#0 [internal function]: PhpOffice\PhpSpreadsheet\Calculation\LookupRef::INDIRECT('R6C1', 0, Object(PhpOffice\PhpSpreadsheet\Cell\Cell))
#1 /vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Calculation/Calculation.php(3777): call_user_func_array(Array, Array)
#2 /vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Calculation/Calculation.php(2754): PhpOffice\PhpSpreadsheet\Calculation\Calculation->processTokenStack(Array, 'AF5', Object(PhpOffice \PhpSpreadsheet\Cell\Cell))
#3 /vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Calculation/Calculation.php(2 in /media/sf_work2/______________________Temp/excel/vendor/phpoffice/phpspreadsheet/src/PhpSpreadsheet/Calculation/ LookupRef.php on line 280
```

```php
<?php

require __DIR__ . '/vendor/autoload.php';

$reader = \PhpOffice\PhpSpreadsheet\IOFactory::createReader('Xlsx');
$reader->setReadDataOnly(TRUE);

$filename = 'XXX.xlsx';
$spreadsheet = $reader->load($filename);
$worksheet = $spreadsheet->getActiveSheet();

foreach ( $worksheet->getRowIterator() as $r => $row ){
	foreach ( $cellIterator as $c => $cell ){
		$val = $cell->getCalculatedValue();
	}
}
```

### Which versions of PhpSpreadsheet and PHP are affected?
PHP 7.0.25-0ubuntu0.16.04.1 (cli) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
with Zend OPcache v7.0.25-0ubuntu0.16.04.1, Copyright (c) 1999-2017, by Zend Technologies

**Comment by sdlins** [comment]
Hi, any corrections or workaround for this?

PHPSpreadSheet `getCalculatedValue()` returns only `#REF!` error when using `Indirect()` refering to other sheets. No problem in other formulas calling other sheets.

**Comment by PieterCappelle** [comment]
Hi, did you find any solution for this issue?

**Comment by oleibman** [comment]
Fixed by PR #1913 in March 2021.
